.travis/install.sh

```diff
@@ -32,10 +32,10 @@ if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
     esac;
 
     source activate test-environment
-    pip install numpy nose coveralls
+    pip install numpy nose coveralls flake8
```

Comment:
Will flake8 only run on travis? What about if we are doing development on our own systems? Do we have to run it manually?

Reply:
Yes. I run it on the fly in my editor.